src/Grid/Grid/Grid.component.tsx

```diff
@@ -51,35 +50,69 @@ type AlignContent =
 
 interface GridProps {
     alignContent?: AlignContent;
-    areas?: [string];
+    areas?: string[];
     children?: any;
     className?: string;
+    columnGap?: string;
     columns?: number | string;
     debug?: boolean;
     flow?: Flow;
     gap?: string;
+    height?: string;
     justifyContent?: Justifycontent;
-    rows?: string;
+    minRowHeight?: string;
+    rowGap?: string;
+    rows?: number | string;
 }
 
-const StyledGrid = styled(AGrid)<GridProps>`
-    ${({ debug }) =>
-        debug
-            ? css`
-                  background-color: rgba(255, 0, 0, 0.4);
-              `
-            : null}
+const frGetter = (value: number | string) =>
+    typeof value === 'number' ? `repeat(${value}, 1fr)` : value;
+
+const formatAreas = (areas: string[]) =>
+    areas.map(area => `'${area}'`).join(' ');
+
+const StyledGrid = styled('div')<GridProps>`
+    display: grid;
+    ${({
+        alignContent,
+        areas,
+        columnGap,
+        columns = 12,
+        debug,
+        flow = FLOW.row,
+        gap = '0.5rem',
+        height = 'auto',
+        justifyContent,
+        minRowHeight = '1.25rem',
+        rowGap,
+        rows,
+    }) => css`
+        height: ${height};
```

Comment:
I'm surprised you didn't use the object format for this. It all shakes out the same in the end, but this seems a lot more verbose. No need to change it—just commenting.

Reply:
No specific reason why I didn't. I'll update for consistency.

Reply:
Actually I take it back. Because most of the properties are optional I feel it's cleaner and simpler this way. I started converting it solely to an object with values and it was becoming much more verbose but for little benefit.